eventkit_cloud/utils/external_service.py

```diff
@@ -110,7 +110,7 @@ def convert(self, ):
             task_process = TaskProcess(task_uid=self.task_uid)
             task_process.start_process(billiard=True, target=seeder.seed,
                         kwargs={"tasks": seed_configuration.seeds(['seed']),
-                                "concurrency": 1,
+                                "concurrency": int(getattr(settings, 'MAPPROXY_CONCURRENCY', 1)),
```

Comment:
maybe we can cast to int pre-emptively in settings?  You also default `MAPPROXY_CONCURRENCY` to 1 so I believe this getattr might not be needed.

imho this is fine to leave as-is :)